### PR TITLE
added map

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+
+    <uses-sdk
+        android:minSdkVersion="16"
+        android:targetSdkVersion="22" />
+
+    <application
+      android:name=".MainApplication"
```
Comment (ovekyc) [inline android/app/src/main/AndroidManifest.xml]: Please change application name

File: shared/actions/actionTypes.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const TBD = 'TBD';
```
Comment (mun625) [inline shared/actions/actionTypes.js]: I think we should set name of function(variable) to more readable.
(can't catch function of TBD at once)

File: android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (ochanje210) [inline android/app/src/main/AndroidManifest.xml]: wrong

File: shared/components/map.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+export default class Map extends Component {
+    render() {
+        return (
+            <View style ={styles.container}>
+                <MapView style ={styles.map}
+                         initialRegion={{
+                             latitude: 37.78825,
+                             longitude: -122.4324,
+                             latitudeDelta: 0.0922,
+                             longitudeDelta: 0.0421,
```
Comment (mun625) [inline shared/components/map.js]: You separate style value to variable. but why didn't you separate initial Region variable??

File: .buckconfig [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+[android]
+  target = Google Inc.:Google APIs:23
```
Comment (festinalente91) [inline .buckconfig]: Is there any reason for using API 23? Because it only cover 4.3% of users ( according to android studio).

Comment (festinalente91) [inline .buckconfig]: Is there any reason for using API 23 ? Because it only covers 4.7% of users. (According to android studio)

**Comment by yuda1124** [comment]
LGTM

File: shared/actions/actionTypes.js (hunk 2 above)
Comment (ochanje210) [inline shared/actions/actionTypes.js]: TBD : to be determined

**Comment by mun625** [comment]
LGTM

File: .buckconfig (hunk 4 above)
Comment (ochanje210) [inline .buckconfig]: great point

**Comment by yuda1124** [comment]
LGTM

**Comment by HewonJeong** [comment]
[As a server administrator, I'd like to correspond data with verified user so that I can protect information](https://trello.com/c/iL3ZBZCQ/170-as-a-server-administrator-i-d-like-to-correspond-data-with-verified-user-so-that-i-can-protect-information)

**Pull request merged**